ansible/roles/ocp4-workload-tl500/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Can you please put this workload under roles_ocp_workloads as that a dedicated directory to OCP Workload kind of roles?

Reply:
done